spec-test-suite.md

```diff
@@ -0,0 +1,107 @@
+# Introduction
+
+The purpose of this index is to provide an overview of the testing approaches for the gossip spec. It is intended to evolve as the framework matures, leaving room for novel cases and extensions of existing cases, as called for by any protocol idiosyncrasies that may come to light during the development process.
+
+Some test cases have been consolidated when similar behaviour is tested with differing messages. The final implementation of these cases will be subject to factors such as node setup and teardown details, test run time (and potentially runtime) constraints, readability and maintainability.
+
+> [!Tip]
+> **Naming conventions used in this document**
+> - _Node_ - a validator running the gossip
+> - _Synthetic node_ - a lightweight gossip node that is used only for testing gossip protocol while interacting with the node via tests.
+> - _Cluster_ - a network of validators with a leader that produces blocks
+
+## Special Considerations
+
+Some of these tests can be executed against the same node instance without resetting or clearing the cache to cut down on node setup and teardown times. Other tests are intended for use where a new peer is needed to clean the slate for a deterministic output.
```

Comment:
_slate_ - should be s**t**ate

Reply:
Actually, I'm gonna remove this chapter for now 😄

Reply:
Done